docs/how-to-docs/production.md

```diff
@@ -0,0 +1,72 @@
+## Production <a name="production"></a>
+
+Using production & setting up production is slightly different to the not-prod cluster, these are the steps:
+
+  1. [Set up a prod token](#prod-token)
+  2. [add the cluster to your kube file](#add-token)
+  3. [add the token to your kube file](#prod-cluster)
+  4. [connect to the prod vpn](#prod-vpn)
+  5. [drone](#prod-drone)
+
+### Set up a prod token <a name="prod-token"></a>
+
+The following steps in this section can only be done by an administrator
+* Go to the [platformhub](https://hub.acp.homeoffice.gov.uk)
+* Click projects -> modern slavery -> details
+* Scroll to the top and click on `KUBE USER TOKENS`
+* And then search for your name in the drop-down
+* And then click `create a new kubernetes user token for this user`
+* Add the user as read-only
+* Escalate privileges to two hours, you'll need to do this everytime you are on.  If you're not an administrator then you'll have to ask your admin
+
+### Add the user token to your kube file <a name="add-token"></a>
+
+* Copy the token you generated above
+* Go to your kube config file, usually located in `~./kube/config`
+* Add the following to your users
+
+```
+  users:
+  - name: acp-prod
```

Comment:
iirc user tokens are scoped to the project in acp? if so you might want to name them more specifically

Reply:
you can do this more easily with
```kubectl config set-context```